torchaudio/models/wavernn.py

```diff
@@ -0,0 +1,103 @@
+from typing import Optional
+
+from torch import Tensor
+from torch import nn
+
+__all__ = ["ResBlock", "MelResNet"]
+
+
+class ResBlock(nn.Module):
+    r"""This is a ResNet block layer. This layer is based on the paper "Deep Residual Learning
+    for Image Recognition". Kaiming He,  Xiangyu Zhang, Shaoqing Ren, Jian Sun. CVPR, 2016.
+    Users may modify or implement in a different way during application. It is a block used in WaveRNN
+    (https://github.com/G-Wang/WaveRNN-Pytorch)
+
+    Args:
+        num_dims: the number of compute dimensions in the input (default=128).
```

Comment:
thinking out loud: we decided in conventions to use `n_*` see [readme](https://github.com/pytorch/audio). What is written here aligns with wav2letter though. We may need to revisit these conventions.

Reply:
#721